FFMpegCore/FFMPEG/FFProbe.cs

```diff
@@ -85,10 +85,9 @@ private VideoInfo ParseVideoInfoInternal(VideoInfo info, string probeOutput)
             else
             {
                 sDuration = (video ?? audio).Tags.Duration;
-                TimeSpan.TryParse(sDuration.Remove(sDuration.LastIndexOf('.')), CultureInfo.InvariantCulture, out duration);
+                TimeSpan.TryParse(sDuration.Remove(sDuration.LastIndexOf('.') + 8), CultureInfo.InvariantCulture, out duration); // TimeSpan fractions only allow up to 7 digits
             }
-            // Strip milliseconds and additional ticks
-            info.Duration = new TimeSpan(duration.Days, duration.Hours, duration.Minutes, duration.Seconds);
+            info.Duration = duration;
```

Comment:
Increased duration resolution and updated the regarding tests.
I think it only makes sense to test the resulting duration up to seconds.
Individual tick differences are to be expected when converting a video.